pisa/core/stage.py

```diff
@@ -325,9 +325,6 @@ def setup_function(self):
         pass
 
     def compute(self):
-        
-        if len(self.params) == 0:
-            return
```

Comment:
why remove this? if a stage has no params it will fail otherwise....or we will need to fix the param hashing for empty params

Reply:
There was at least one stage that relied on the behavior that the compute function was at least executed once, even if the stage has no parameters. Was there a problem with the hashing for empty params?

Reply:
I think remembering running into problems....but maybe it's ok